torq.q

```diff
@@ -460,11 +460,16 @@ if[not any `debug`noredirect in key params; rolllogauto[]];
 
 // utilities to load individual files / paths, and also a complete directory
 // this should then be enough to bootstrap
-loadf:{
-	.lg.o[`fileload;"loading ",x];
+loadedf:enlist enlist""
+loadf0:{[reload;x]
+    if[not[reload]&x in loadedf;.lg.o[`fileload;"already loaded ",x];:()];
+    loadedf,:enlist x;
```

Comment:
File will be marked as loaded even if line 469 fails, is that intended?

Reply:
good point